### PR TITLE
feat: add mkdir command

### PR DESCRIPTION
**Pull request by sigmaSd**
I run `drpint fmt` still same issue, is there something I need to do ?

### CODE REVIEW

**Comment by sigmaSd** [comment]
It seems like you're not using the default dprint config ?, but https://github.com/dsherret/dax/blob/main/dprint.json doesn't have any configuration

**Comment by dsherret** [comment]
dprint leaves a lot of code as-is, so I think maybe something else is formatting the files while you're working on them? The default width for dprint is 120 characters for example, but it seems like something else is formatting the files.

Would you be able to revert all formatting changes with git?

**Comment by dsherret** [comment]
I'll align dprint with deno fmt after we merge this PR.

File: mod.test.ts
Comment (dsherret) [inline mod.test.ts:741]: In #40 I just added a withTempDir helper. Let's use that here to ensure that these temporary directories get cleaned up after the test is run.

File: src/commands/mkdir.ts
Comment (dsherret) [inline src/commands/mkdir.ts:19]: Let's change this to start with a capital: `MkdirFlags`

File: mod.test.ts
Comment (dsherret) [inline mod.test.ts:760]: Nit, typo: expectedError

File: src/utils.ts
Comment (dsherret) [inline src/utils.ts:1]: Maybe call this `lstat` since it's using `Deno.lstat` and not `Deno.stat`?

File: src/utils.ts
Comment (dsherret) [inline src/utils.ts:7]: Maybe we should just return `Deno.FileInfo | undefined` from this function? Then we could do a stat call once in the code above. Right now we're doing possibly two stat calls per path instead of max 1.

**Review by dsherret — COMMENTED** [review]
Thanks for adding all these commands, @sigmaSd! We're almost there on this one.

File: src/utils.ts
Comment (sigmaSd) [reply src/utils.ts]: Actually I see your point, can you merge this as is (if there are no other issues), and I'll do the changes in the other pr

Comment (dsherret) [reply src/utils.ts]: Sure.

**Review by dsherret — APPROVED** [review]
Looks great. Thanks!

**Pull request merged**
